eth/examples/SKALE/bountiesEarned.ts

```diff
@@ -1,5 +1,5 @@
 import { BitGo } from 'bitgo';
-import { Contract } from '../../src/contract';
+import { getContractsFactory } from '../../../src/index';
```

Comment:
Not a huge fan of this naming - maybe `contractsFactory` is better?

Reply:
If we use that type of naming (property based), it will get confuse as if you call like this `contractFactory('eth')` that reads as if you are setting a factory not getting one.